openupgrade_scripts/scripts/account_edi/14.0.1.0/upgrade_analysis.txt

```diff
@@ -20,8 +20,6 @@ NEW ir.model.access: account_edi.access_account_edi_document_group_invoice
 NEW ir.model.access: account_edi.access_account_edi_document_readonly
 NEW ir.model.access: account_edi.access_account_edi_format_group_invoice
 NEW ir.model.access: account_edi.access_account_edi_format_readonly
-NEW ir.model.constraint: account_edi.constraint_account_edi_document_unique_edi_document_by_move_by_format
```

Comment:
Why did you removed the lines ?

Reply:
> Why did you removed the lines ?

@legalsylvain Because I didn't notice. I fixed it already. plz review :D